zerver/models.py

```diff
@@ -1256,6 +1256,8 @@ class RealmFilter(models.Model):
     realm = models.ForeignKey(Realm, on_delete=CASCADE)
     pattern = models.TextField()
     url_template = models.TextField(validators=[url_template_validator])
+    # Order that each linkifier is applied when they have overlapping patterns.
```

Comment:
I would frame this as something like "# Linkifiers are applied in a message in order; the processing order is important when there are overlapping patterns." -- the framing you have makes it sound like there's a check for whether patterns overlap, which isn't exactly the right framing.